subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/NotificationServiceTests.kt

```diff
@@ -212,15 +257,17 @@ class NotificationServiceTests {
         every { subscriptionService.updateSubscriptionNotification(any(), any(), any()) } answers { Mono.just(1) }
         every { notificationsEventsListener.handleNotificationEvent(any()) } just Runs
 
-        stubFor(post(urlMatching("/notification"))
-            .willReturn(ok()))
+        stubFor(
+            post(urlMatching("/notification"))
+                .willReturn(ok())
+        )
 
         StepVerifier.create(notificationService.callSubscriber(subscription, listOf(parsedEntity)))
             .expectNextMatches {
                 it.first.id == subscription.id &&
-                it.second.subscriptionId == subscription.id &&
```

Comment:
indentation seems to be not good here (and below).

Reply:
This is to clearly identify the continuation of the expression, so you clearly see at first sight that this is all a single expression.

Maybe in this case it seems useless, but you could have cases like that : 
```kotlin
val a = it.size == 1 &&
it[0].second.subscriptionId == subscription.id &&
it[0].second.data.size == 1 &&
it[0].third
a
```
There it becomes more relevant to have this indent to more easily detect the what is variable a and what is returned.